tests/vptr/accessor.cc

```diff
@@ -121,9 +121,8 @@ TEST(accessor, two_buffers) {
   }
 }
 
+// Is this test still required, given that we no longer use allocators?
```

Comment:
I'm not sure if this test is still required, as as far as I can tell, all it does is test that we've actually added two buffers?

Reply:
No, Thanks for fixing it